src/overlays/actors/ovl_En_Viewer/z_en_viewer.h

```diff
@@ -38,9 +65,9 @@ typedef struct EnViewer {
     /* 0x01DD */ u8 drawFuncIndex;
     /* 0x01E0 */ EnViewerActionFunc actionFunc;
     /* 0x01E4 */ u8 unused;
-    /* 0x01E5 */ u8 unk_1E5;
-    /* 0x01E6 */ u8 unk_1E6; // Conditional
-    /* 0x01E8 */ EnViewerUnkStruct unk_1E8[20];
+    /* 0x01E5 */ u8 state;
+    /* 0x01E6 */ u8 draw;
```

Comment:
a name that makes it more clear that this is a bool like `isVisible` could be desirable